src/SvgParser.elm

```diff
@@ -216,7 +218,12 @@ nodeParser =
 -}
 toAttribute : SvgAttribute -> Attribute msg
 toAttribute ( name, value ) =
-    attribute name value
+    if name == "xlink:href" then
```

Comment:
Thanks.

Reply:
I checked the source of `Svg.xlinkHref`:

https://github.com/elm/svg/blob/dfe97e8282f283f4a62b6645f633076cfd24f3f7/src/Svg/Attributes.elm#L1183-L1185

It seems like that we need namespaced xml attributes to make this work. I will create a general fix for this later. For the time being your fix is okay.